lib/active_job/json_log_subscriber.rb

```diff
@@ -0,0 +1,267 @@
+# frozen_string_literal: true
+
+require "active_support/subscriber"
+require "active_job/log_subscriber"
+
+module ActiveJob
+  class JsonLogSubscriber < ActiveSupport::LogSubscriber # :nodoc:
+    # rubocop:disable ThreadSafety/ClassAndModuleAttributes
+    class_attribute :backtrace_cleaner, default: ActiveSupport::BacktraceCleaner.new
+    # rubocop:enable ThreadSafety/ClassAndModuleAttributes
+
+    def enqueue(event)
+      job = event.payload[:job]
+      ex = event.payload[:exception_object] || job.enqueue_error
+
+      if ex
+        error do
+          {
+            event: "enqueue",
+            status: "error",
+            job: job.class.name,
+            queue: job.queue_name,
```

Comment:
Is there a reason to not have the arguments here? 🤔 
and for other logs on error

```suggestion
            queue: job.queue_name,
            arguments: args_info(job)
```

Reply:
yes! we have the dead job for this! btw default AJ logger does not provides arguments on error